Doc/library/enum.rst

```diff
@@ -161,13 +161,13 @@ Data Types
 
 .. class:: EnumType
 
-   *EnumType* is the :term:`metaclass` for *enum* enumerations.  It is possible
+   :class:`EnumType` is the :term:`metaclass` for :class:`enum` enumerations.  It is possible
```

Comment:
Linking `EnumType` here is ugly as the link would lead right back to the current paragraph. We probably should use code formatting, but not a link.

`enum` I believe refers to the module, not to a class—there is no class called `enum`. There too, I don't think a link is helpful.